ccmlib/node.py

```diff
@@ -1254,22 +1257,34 @@ def _update_pid(self, process):
             raise NodeError('Problem starting node %s' % self.name, process)
         self.__update_status()
 
-    def __gather_sstables(self, datafile=None, keyspace=None, columnfamilies=None):
-        datafiles = []
+    def __gather_sstables(self, datafiles=None, keyspace=None, columnfamilies=None):
+        files = []
         if keyspace is None:
-            for k in self.list_keyspaces():
-                datafiles = datafiles + self.get_sstables(k, "")
-        elif datafile is None:
+            if datafiles:
+                for datafile in datafiles:
+                    if not os.path.isabs(datafile):
+                        datafile = os.path.join(os.getcwd(), datafile)
+
+                    if not os.path.exists(datafile):
+                        print_("File doesn't exist: " + datafile)
+                    elif datafile.endswith('Data.db') or datafile.endswith('big-Data.db'):
```

Comment:
thinking more about that, this will not have the same behavior if the user call the split with in example \*Filter.db,  Which is valid directly through sstablesplit. The best option would be to track the sstable number and only add one file for each. Let me know if this sounds like a reasonable approach.